src/frontend/src/test-e2e/selenium.test.ts

```diff
@@ -37,10 +37,11 @@ import canister_ids2 from "../../../../demos/whoami/.dfx/local/canister_ids.json
 const IDENTITY_CANISTER = canister_ids1.internet_identity.local;
 const WHOAMI_CANISTER = canister_ids2.whoami.local;
 
-const REPLICA_URL = "http://localhost:8000";
-const II_URL = `http://localhost:8000/?canisterId=${IDENTITY_CANISTER}`;
-const FAQ_URL = `http://localhost:8000/faq?canisterId=${IDENTITY_CANISTER}`;
-const ABOUT_URL = `http://localhost:8000/about?canisterId=${IDENTITY_CANISTER}`;
+const REPLICA_URL = "https://ic0.app";
```

Comment:
That's gonna break the tests locally unfortunately. Can you make this conditional somehow?

Reply:
Can we keep it for now? We'll move to docker-compose ASAP anyway.

Reply:
Added a switch